### ISSUE TITLE
sphinx 3 support

### ISSUE THREAD

**Issue by oliver-sanders** [nyergler/hieroglyph#166]
From the Sphinx [changelog](https://github.com/sphinx-doc/sphinx/blob/713bbf5cafa3fc5e143ced59dafe56f4b802ef80/CHANGES#L61):

> Drop features and APIs deprecated in 1.8.x

The current behaviour:

```console
$ pip list | grep hieroglyph
hieroglyph                     1.0.0   

$ sphinx-build --version
sphinx-build 3.0.0

$ python -c 'import hieroglyph'
ImportError: cannot import name 'copy_static_entry' from 'sphinx.util' (python3.7/site-packages/sphinx/util/__init__.py)
```

**Comment by ajaeger** [comment]
This looks to be fixed with https://github.com/nyergler/hieroglyph/pull/163 - so, we needs a new release of hierglyph.

**Comment by nyergler** [comment]
I just released Hieroglyph 2.0 on PyPI; the tests for it pass with Sphinx 2.x and 3.x, so I expect this issue has been resolved.

Apologies for the delay in getting a release out; in an effort to streamline the next one, I added documentation for myself (or others!) for making releases.

Thanks!

**Comment by lpirl** [comment]
Great, thanks @nyergler – I'll report back if the problem persists.
